src/execution_plan/execution_plan_build/build_call_subquery.c

```diff
@@ -296,3 +287,4 @@ void buildCallSubqueryPlan
 		ExecutionPlan_Free(embedded_plan);
 	}
 }
+
```

Comment:
line 283 `ExecutionPlan_AddOp(call_op, embedded_plan->root);` not sure if that's true,
we've migrated the root of the inner plan to `plan` I don't see a reason why are we connecting the `call` op to any operation within the `embedded` plan.